src/shared/components/annotation/OncoKB.tsx

```diff
@@ -17,6 +17,7 @@ import OncoKbEvidenceCache from "shared/cache/OncoKbEvidenceCache";
 import OncoKbTooltip from "./OncoKbTooltip";
 import OncokbPubMedCache from "shared/cache/PubMedCache";
 import {default as TableCellStatusIndicator, TableCellStatus} from "shared/components/TableCellStatus";
+import AppConfig from "appConfig";
```

Comment:
I think it is better to make `userEmailAddress` a part of `IOncoKbProps`. In general we should try to avoid "injecting" `AppConfig` into the shared components, since it will cause problems when we want to separate certain components out of this repository.

Reply:
Only if other parts will not use this property. I'm so sure about all features.